Makefile

```
@@ -1,11 +1,11 @@
 build:
-	cargo build --release 
+	cargo build --release
 
 install: build
-	sudo cp target/release/miro /usr/local/bin
+	cp target/release/miro /usr/local/bin
```

```suggestion
	cargo install --path "."
```
Like that we won't need root privileges.